_posts/2016-12-22-Rust-1.14.md

```diff
@@ -0,0 +1,356 @@
+
+[numer of improvements to compile times]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#compile-time-optimizations
+
+In the language, one small improvement has landed: support for [RFC 1492]. This small
+additional lets you use `..` in more places. Previously, say you had a struct like this:
+
+```rust
+struct Point {
+    x: i32,
+    y: i32,
+    z: i32,
+}
+```
+
+In any context where you're doing a pattern match, you could use `..` to ignore the
+parts you don't care about. For example:
+
+
+```rust
+let p = Point { x: 0, y: 1, z: 2 };
+
+match p {
+    Point { x, .. } => println!("x is {}", x),
+}
+```
+
+The `..` ignores `y` and `z`.
+
+Consider a similar `Point`, but as a tuple struct:
+
+```rust
+struct Point(i32, i32, i32);
+```
+
+Previously, you could use `..` to ignore all three elements:
+
+```rust
+let p = Point(0, 1, 2);
+
+match p {
+    Point(..) => println!("found a point"),
+}
+```
+
+But you could not use it to only ignore parts of the tuple:
+
+```rust
+let p = Point(0, 1, 2);
+
+match p {
+    Point(x, ..) => println!("x is {}", x),
+}
+```
+
+This was a inconsistency, and so with RFC 1492 stabilized, compiles fine as of
+this release. This applies to more situations than tuples; please see [the
+RFC][RFC 1492] for more details.
+
+[RFC 1492]: https://github.com/rust-lang/rfcs/blob/master/text/1492-dotdot-in-patterns.md
+
+#### Library stabilizations
+
+There have been a number of additions to the standard library, but they don't
+fit into particularly nice categories for this release. Here's the highlights:
+
+* [`println!()`, with no arguments, prints newline][36825].
+  Previously, an empty string was required to achieve the same.
+* [`Wrapping` impls standard binary and unary operators, as well as
+   the `Sum` and `Product` iterators][37356]
+* [Implement `From<Cow<str>> for String` and `From<Cow<[T]>> for
+  Vec<T>`][37326]
+* [Expand `.zip()` specialization to `.map()` and `.cloned()`][37230]
+* [Implement `RefUnwindSafe` for atomic types][37178]
+* [Specialize `Vec::extend` to `Vec::extend_from_slice`][37094]
+* [Don't reuse `HashMap` random seeds][37470]
+* [The internal memory layout of `HashMap` is more cache-friendly, for
+  significant improvements in some operations][36692]
+* [Impl `Add<{str, Cow<str>}>` for `Cow<str>`][36430]
+
+[36825]: https://github.com/rust-lang/rust/issues/36825
+[37356]: https://github.com/rust-lang/rust/issues/37356
+[37326]: https://github.com/rust-lang/rust/issues/37326
+[37230]: https://github.com/rust-lang/rust/issues/37230
+[37178]: https://github.com/rust-lang/rust/issues/37178
+[37094]: https://github.com/rust-lang/rust/issues/37094
+[37470]: https://github.com/rust-lang/rust/issues/37470
+[36692]: https://github.com/rust-lang/rust/issues/36692
+[36430]: https://github.com/rust-lang/rust/issues/36430
+
+See the [detailed release notes][notes] for more.
+
+#### Cargo features
+
+As for Cargo, [RFC 1721] has been implemented. The motivation for this feature
```

Comment:
You say "the motivation for this feature" without actually saying what the feature is.